packages/genesys-spark-tokens/data/ui/forms_and_inputs.json

```diff
@@ -1069,6 +1069,190 @@
       }
     }
   },
+  "file_upload": {
+    "file_card": {
+      "border_radius": {
+        "value": "{interactive.sm.border_radius}",
+        "type": "borderRadius"
+      },
+      "card_group": {
+        "gap": {
+          "value": "{container.sm.gap}",
+          "type": "spacing"
+        }
+      },
+      "file_name": {
+        "text": {
+          "value": "{body.sm.regular}",
+          "type": "typography"
+        },
+        "default": {
+          "value": "{foreground.form_control.text_input.populated}",
+          "type": "color"
+        }
+      },
+      "error": {
+        "error_message": {
+          "value": "{foreground.form_control.help_text.error}",
+          "type": "color"
+        },
+        "error_helper": {
+          "value": "{foreground.form_control.help_text.default}",
+          "type": "color"
+        },
+        "text": {
+          "value": "{body.sm.regular}",
+          "type": "typography"
+        }
+      },
+      "card": {
+        "padding": {
+          "value": "{container.2xs.padding} {container.xs.padding}",
+          "type": "spacing"
+        },
+        "gap": {
+          "value": "{container.xs.gap}",
+          "type": "spacing"
+        },
+        "error_card": {
+          "file_name_section": {
+            "padding": {
+              "value": "0 {container.xs.padding}",
+              "type": "spacing"
+            }
+          },
+          "error_text_section": {
+            "padding": {
+              "value": "0 {container.xs.padding}",
+              "type": "spacing"
+            },
+            "gap": {
+              "value": "{container.xs.gap}",
+              "type": "spacing"
+            }
+          },
+          "main_container": {
+            "padding": {
+              "value": "{interactive.sm.padding} 0 {interactive.md.padding}",
```

Comment:
I was just wondering are these `0` values okay ?

Reply:
I think they are for now but maybe it is something we should review and come up with an official policy. What values do not need to be tokens?